packages/combinator/src/assets/themes/light.less

```diff
@@ -1,11 +1,11 @@
 & {
-  --dtc-theme-color-foreground: var(--black-800);
-  --dtc-theme-color-foreground-darken: var(--black-900);
-  --dtc-theme-color-foreground-lighten: var(--black-700);
+  --dtc-theme-color-foreground: var(--dt-color-black-800);
+  --dtc-theme-color-foreground-darken: var(--dt-color-black-900);
+  --dtc-theme-color-foreground-lighten: var(--dt-color-black-700);
 
-  --dtc-theme-color-background: var(--white);
+  --dtc-theme-color-background: var(--dt-color-neutral-white);
   --dtc-theme-color-background-darken: var(--black-050);
-  --dtc-theme-color-background-lighten: var(--white);
+  --dtc-theme-color-background-lighten: var(--dt-color-neutral-white);
 
   color-scheme: light;
 }
```

Comment:
This of course needs to be updated to use proper dialtone theming now. Not a big priority though.

Reply:
Yes, I will defer that for a later stage when Design gives us something to start working with.